src/sdk/namespace_fs.js

```diff
@@ -39,6 +39,9 @@ const XATTR_USER_PREFIX = 'user.';
 // TODO: In order to verify validity add content_md5_mtime as well
 const XATTR_CONTENT_TYPE = XATTR_USER_PREFIX + 'content_type';
 const XATTR_MD5_KEY = XATTR_USER_PREFIX + 'content_md5';
+const XATTR_PART_OFFSET = XATTR_USER_PREFIX + 'part_offset';
+const XATTR_PART_SIZE = XATTR_USER_PREFIX + 'part_size';
+const XATTR_PART_ETAG = XATTR_USER_PREFIX + 'part_etag';
```

Comment:
add noobaa prefix - user.noobaa.xattrname
we want for the internal xattrs, i.e ones that we do not want others to rely on, we want to add the noobaa prefix.